src/main/kotlin/no/nav/familie/oppdrag/service/AvstemmingService.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.familie.oppdrag.service
+
+import no.nav.familie.oppdrag.avstemming.AvstemmingSender
+import no.nav.familie.oppdrag.repository.OppdragProtokollRepository
+import no.nav.virksomhet.tjenester.avstemming.meldinger.v1.Avstemmingsdata
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+
+@Service
+class AvstemmingService(
+        @Autowired private val avstemmingSender: AvstemmingSender,
+        @Autowired private val oppdragProtokollRepository: OppdragProtokollRepository) {
+
+    fun utførGrensesnittavstemming(listeAvstemmingMeldinger: List<Avstemmingsdata>) {
```

Comment:
Her _kan_ det jo vurderes om funksjonaliteten for å hente ut avstemmningsdata fra databasen og gjøre mapping kan flyttes inn her, i stedet for å skje i controlleren. Slik at signaturen da blir ca :
`fun utførGrensesnittavstemming(fomDato, tomDato,fagomrdåer)`

Det vil gjøre controlleren "dummere" og kan testes uavhengig av MQ og DB, og service-metoden kan gjøres transaksjonell, hvis ønskelig.